### ISSUE TITLE
get_remote_name does not trim output

### ISSUE THREAD

**Issue by sinistersnare** [arusahni/git-req#79]
Hi, I changed the remote name, and `git-req` prompted me to choose a new default remote.

```
Multiple remotes detected. Enter the name of the default one.
 * davis
 * upstream
Remote name: upstream
The application panicked (crashed).
Message:  Couldn't find the remote: Error { code: -12, klass: 7, message: "'upstream\n' is not a valid remote name." }
Location: src/git.rs:40
```

I think it is because you need to [`#trim()`](https://doc.rust-lang.org/src/core/str/mod.rs.html#1867) the output of the `stdin` [here](https://github.com/arusahni/git-req/blob/master/src/main.rs#L176)

**Comment by arusahni** [comment]
Thanks for the report! I've merged a fix and will cut a release sometime this weekend.

**Comment by arusahni** [comment]
This fix is part of the 2.5.0 release.
